Tasks/AzureMysqlDeployment/operations/ToolPathOperations.ts

```diff
@@ -0,0 +1,136 @@
+import task = require('vsts-task-lib/task');
+var winreg = require('winreg');
+import Q = require('q');
+
+export class ToolPathOperations {
```

Comment:
@Ajay-MS - Can you review this to ensure we are covered based on our learnings with other SQL task.